### ISSUE TITLE
v111.5, Commit 45e9e0 - Battletoads & Double Dragon (USA) Freeze

### ISSUE THREAD

**Issue by Pr3ttyF1y** [byuu/bsnes#165]
With the Fast PPU on, Battletoads & Double Dragon (USA) freezes when you break open the first item in Stage 1 that produces a 1-Up (i.e., at the very beginning of the 1st stage after the first 2 bad guys).

With Fast PPU off, the game freezes at the title screen with corrupt line below the title graphics.

The SPC-700 music continues to play after the gameplay freezes and the BSNES.exe itself locks-up and needs.

**Comment by Max833** [comment]
> Disable rewind (settings -> emulator -> rewind -> frequency) or load the latest version v111r5 
https://github.com/byuu/bsnes/commits/master) and it should work. 

https://github.com/byuu/bsnes/issues/160

**Comment by jjfmz42015** [comment]
@byuu the default settings.bml on your releases has rewind on by default. Future releases I think disable it. More user friendly for those that wouldn't toggle it and wouldn't suspect it causing trouble

**Comment by jjfmz42015** [comment]
Ide also add warning enabling can cause unexpected behavior leave disabled for better compatibility

**Comment by byuu** [comment]
It should have rewind disabled in default releases, but okay if I did make that mistake then I'll go and fix it.

**Comment by jjfmz42015** [comment]
I wouldn't call it a mistake. I'm only giving some feedback as an end user in hope to be helpful.

**Comment by Max833** [comment]
> The goal of this thread is to track games that challenge various serialization methods.
> 
> The end goal will likely be the most reliable method being the default, with per-game overrides to choose different methods for the rest. 
> 
> https://board.byuu.org/viewtopic.php?f=4&p=16&sid=73064055fa07215bbc33393cb9973d00#p16

@byuu: I can already answer this. The first rewind function was the most reliable. I played 1000 games with it and I've never had problems. Star Ocean was the first game that freezes. With the latest method I already have 5 games out of 70 that freezes.

**Comment by byuu** [comment]
Max: okay, I'll make that method the default.

**Comment by Max833** [comment]
I've tested v111.6 and it's way better than before, but "Lost Vikings" has still the same issue (https://github.com/byuu/bsnes/issues/153) and when you watch the attract mode of "NBA Hang Time", "NBA Jam - Tournament Edition" or "NCAA Football" and press rewind/forward, it freezes. I thought that this is really odd, because I've played 1000 games without problems before and now after 5-10 games this happens. And yeah... it turns out that there are no problems with v111. Is this really the first rewind methode?
